### ISSUE TITLE
Refresh thumbnail after rotate

### ISSUE THREAD

**Issue by AlbatorV** [SimpleMobileTools/Simple-Gallery#2284]
Hi,
When I edit image from editor to perform rotation, we can save file and choose name.
Often, I want to replace the original image, so I delete the prefix "_1" and save to overwrite file.
Problem, thumbnail is not refresh in browse view.
Regards

**Comment by Aga-C** [comment]
It also occurs without entering the edit mode, just rotating on photo's grid. Just after rotating, the thumbnail is also rotated, but when I go back to folder's view and again to previous view, there is an old thumbnail. Also, if the rotated photo is first in the folder, it won't show up as rotated on folder's view.

**Comment by dmint789** [comment]
Yes, please fix this bug. The only way to fix the problem currently is closing the app and reopening it.

**Comment by camoz** [comment]
Somehow even reopening the app doesn't fix it for me.
